File: clang-tools-extra/test/clang-tidy/basic.cpp
// RUN: clang-tidy %s -checks='-*,llvm-namespace-comment' -- | FileCheck %s
// RUN: c-index-test -test-load-source-reparse 2 all %s -Xclang -add-plugin -Xclang clang-tidy -Xclang -plugin-arg-clang-tidy -Xclang -checks='-*,llvm-namespace-comment' 2>&1 | FileCheck %s

namespace i {
}
// CHECK: warning: namespace 'i' not terminated with a closing comment [llvm-namespace-comment]

File: llgo/test/execution/closures/basic.go
// RUN: llgo -o %t %s
// RUN: %t 2>&1 | FileCheck %s

// CHECK: abc

package main

func cat(a, b string) func(string) string {
	return func(c string) string { return a + b + c }
}

func main() {
	f := cat("a", "b")
	println(f("c"))
}
